### DISCUSSION TITLE
When we can get the Vite driver 100% working?

### DISCUSSION THREAD

**Discussion by iyashpal** [adonisjs/core#3991] [Help] [answered]
Most recent [release](https://github.com/adonisjs/core/releases/tag/v5.9.0) says the framework now supports Vite driver but not 100% working. Is there any idea when we can get the Vite driver 100% working?

**Comment by Barbapapazes** [comment]
@Julien-R44 can answer I think!

**Comment by Julien-R44** [comment, marked as answer]
Soon™ 😁

More seriously, it's supposed to be very soon. In fact, 95% of the work was done a month ago, but we were missing some small details that we could not decide on. We finally made a decision, but it came at the wrong time because I'm undergoing several weeks of rush at my "real" professional job, which prevented me from making the necessary changes. 

This weekend I should be able to finish the last modifications. Then it will take a few more days to wait for Virk's reviews. So release very soon !

I'm super glad to see that many of you are waiting for the release ! 🙌

**Reply by Barbapapazes** [reply]
Hooo it looks good as a planning

**Reply by iyashpal** [reply]
@Julien-R44 is there any way to setup vite manually on an existing project before the official release? It would be great if you provide some guidelines. The project I am working on is entering to design implementation phase and I don't want to use the webpack Encore.
